src/github3/__about__.py

```diff
@@ -5,7 +5,7 @@
 __author_email__ = "graffatcolmingov@gmail.com"
 __license__ = "Modified BSD"
 __copyright__ = "Copyright 2012-2018 Ian Stapleton Cordasco"
-__version__ = "1.3.0"
+__version__ = "1.4.0"
```

Comment:
Should probably be 2.0

Reply:
@sigmavirus24 Changed.

Reply:
@sigmavirus24 maybe removing all deprecation warnings and deprecated functions would be a nice idea before launching 2.0.0 then.